### ISSUE TITLE
Animation as the destination view loads glitches

### ISSUE THREAD

**Issue by adku** [rjeitani/testNavBarIssue#1]
Hello,
Did you resolve this issue?
https://developer.apple.com/forums/thread/711302

Right now I'm fighting with it 🗡️ 

Best, 
Adrian

**Comment by rjeitani** [comment]
No, I ended up just re-writing that view in SwiftUI. But someone just responded to the question on the [Stack Overflow question](https://stackoverflow.com/questions/72942907/when-loading-a-swiftui-view-from-a-uikit-view-that-has-a-navigationcontroller-t) I asked, might be worth looking into that.
